mxfusion/components/variables/var_trans.py

```diff
@@ -84,3 +85,47 @@ def __init__(self):
         Initializes as Softplus transformation with 0 offset.
         """
         super(PositiveTransformation, self).__init__(offset=0.)
+
+
+class Logistic(VariableTransformation):
+    """
+    Transformation to constraint a variable to lie between two values.
+    """
+    def __init__(self, lower, upper):
+        """
+        :param lower: Lower bound
+        :param upper: Upper bound
+        """
+        if lower >= upper:
+            raise ValueError('The lower bound is above the upper bound')
+        self._lower, self._upper = lower, upper
+        self._difference = self._upper - self._lower
+
+    def transform(self, var, F=None, dtype=None):
+        """
+        Forward transformation.
+
+        :param var: Variable to be transformed.
+        :type var: mx.ndarray or mx.sym
+        :param F: Mode to run MxNet in.
+        :type F: mxnet.ndarray or mxnet.symbol
+        :param dtype: data type.
+        :type dtype: e.g. np.float32
+        """
+        F = get_default_MXNet_mode() if F is None else F
+        return self._lower + self._difference / (1. + F.exp(-var))
```

Comment:
Could you please use the activation operator of mxnet instead? (mxnet.ndarray.Activation(data, act_type='sigmoid)

This straight implementation has numerical issue when var goes too big.